src/relion/zocalo/tomo_align.py

```diff
@@ -240,6 +240,23 @@ def _tilt(file_list):
                 f"AreTomo failed with exitcode {aretomo_result.returncode}:\n"
                 + aretomo_result.stderr.decode("utf8", "replace")
             )
+            # Update failure processing status
+            if isinstance(rw, RW_mock):
+                rw.transport.send(
+                    destination="failure",
+                    message={
+                        "status": "failure",
+                        "content": {"dummy": "dummy"},
+                    },
+                )
+            else:
+                rw.send_to(
+                    "failure",
+                    {
+                        "status": "failure",
+                        "content": {"dummy": "dummy"},
```

Comment:
I think the second argument of `send_to` can just be an empty string like here: https://github.com/DiamondLightSource/python-zocalo/blob/2029a77b3956e3932e3a7207010d8622301842df/src/zocalo/wrapper.py#L40. All of the stuff required for the message will b in the recipe